xyz.eclipium.Hieroctive.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "xyz.eclipium.Hieroctive",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "Hieroctive",
+    "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
```

Comment:
does it really support wayland?

Reply:
Yes

Reply:
It does indeed seem to work. Doesn't provide any window decorations if required but that's not a blocker for a game given the expected full screen context of use.